### PR TITLE
style(conventions): add strictphp/conventions

### PR DESCRIPTION
**Pull request by h4kuna**
Wait for add rule https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/control_structure/trailing_comma_in_multiline.rst

### CODE REVIEW

File: src/Clients/Event/Events/AbstractRequestEvent.php [hunk 1]
```diff
@@ -10,7 +10,9 @@
 abstract class AbstractRequestEvent
 {
     public readonly string $id;
+
```
Comment (pionl) [inline src/Clients/Event/Events/AbstractRequestEvent.php]: @h4kuna Do you like this? I think i would prefer no line?

File: src/Managers/ConfigManager.php [hunk 2]
```diff
@@ -53,11 +42,23 @@ public function addDefault(ConfigContract $config): void
...
         return $config;
     }
 
+    /**
+     * @param class-string<ConfigContract> $class
+     */
+    private function getDefault(string $class): ConfigContract
+    {
+        if ($this->existsDefault($class) === false) {
+            $this->addDefaultForce(new $class());
```
Comment (pionl) [inline src/Managers/ConfigManager.php]: I would rename `addDefaultSorce ` to forceDeffault

File: src/Managers/ConfigManager.php [hunk 3]
```diff
@@ -53,11 +42,23 @@ public function addDefault(ConfigContract $config): void
...
 
         return $config;
     }
 
+    /**
+     * @param class-string<ConfigContract> $class
+     */
+    private function getDefault(string $class): ConfigContract
+    {
+        if ($this->existsDefault($class) === false) {
```
Comment (pionl) [inline src/Managers/ConfigManager.php]: I would rename `existsDefault` to `defaultExists`.

Make more sense in english

**Review by pionl — COMMENTED** [review]
Seems OK, could you update the strict conventions with proposed rule?

File: src/Managers/ConfigManager.php (hunk 2 above)
Comment (h4kuna) [reply src/Managers/ConfigManager.php]: ok

File: src/Managers/ConfigManager.php (hunk 3 above)
Comment (h4kuna) [reply src/Managers/ConfigManager.php]: ok

File: src/Clients/Event/Events/AbstractRequestEvent.php (hunk 1 above)
Comment (h4kuna) [reply src/Clients/Event/Events/AbstractRequestEvent.php]: As you wish, I have no opinion.

Ok, I will find rule

File: src/Managers/ConfigManager.php (hunk 2 above)
Comment (h4kuna) [reply src/Managers/ConfigManager.php]: see 2f7b3042a6d193482ee25f32e3b58002effdf7d8

**Comment by pionl** [comment]
New version of conventions - v1.0.1

**Comment by h4kuna** [comment]
Add new rules https://github.com/strictphp/conventions/pull/2

**Comment by pionl** [comment]
https://github.com/strictphp/conventions/releases/tag/v1.0.2

**Comment by h4kuna** [comment]
done

**Comment by pionl** [comment]
Thank you

File: composer.json [hunk 4]
```diff
@@ -33,11 +33,27 @@
   },
   "require-dev": {
     "phpstan/phpstan": "^1.8",
+    "phpstan/phpstan-deprecation-rules": "^1.1.3",
```
Comment (pionl) [inline composer.json:36]: should we add this to the base strictphp/conventions?

File: composer.json [hunk 5]
```diff
@@ -33,11 +33,27 @@
...
     "phpstan/phpstan-strict-rules": "^1.4",
-    "phpstan/phpstan-deprecation-rules": "^1.1.3"
+    "strictphp/conventions": "^1.0"
```
Comment (pionl) [inline composer.json:38]: this can break CI with `--prefer-lower` installation

**Review by pionl — APPROVED** [review]

**Pull request merged**